*** spl-0.7.13-orig/scripts/kmodtool	2019-02-22 09:55:48.000000000 -0800
--- spl-0.7.13/scripts/kmodtool	2019-08-27 10:28:36.726967228 -0700
***************
*** 153,160 ****
  Provides:         kmod-${kmodname}-uname-r = ${kernel_uname_r}
  Provides:         ${kmodname}-kmod = %{?epoch:%{epoch}:}%{version}-%{release}
  Requires:         ${kmodname}-kmod-common >= %{?epoch:%{epoch}:}%{version}
! Requires(post):   ${prefix}/sbin/depmod
! Requires(postun): ${prefix}/sbin/depmod
  EOF
  
  	if [[ ${obsolete_name} ]]; then
--- 153,161 ----
  Provides:         kmod-${kmodname}-uname-r = ${kernel_uname_r}
  Provides:         ${kmodname}-kmod = %{?epoch:%{epoch}:}%{version}-%{release}
  Requires:         ${kmodname}-kmod-common >= %{?epoch:%{epoch}:}%{version}
! #Requires(post):   ${prefix}/sbin/depmod
! #Requires(postun): ${prefix}/sbin/depmod
! AutoReq:          no
  EOF
  
  	if [[ ${obsolete_name} ]]; then
